icepyx/tests/test_OpenAltimetry.py

```diff
@@ -0,0 +1,22 @@
+import icepyx as ipx
+from icepyx.core.visualization import Visualize, filelist_latestcycle
+
+
+def test_OA_request():
+    
+    short_name = 'ATL06'
+    date_range = ['2019-4-14','2019-7-1']
+    bbox = [2, -73, 10, -70]
+    region = ipx.Query(short_name, bbox, date_range)
+    filelist_nsidc = region.avail_granules(ids=True, cycles=True, tracks=True)[0]
+    product = region.dataset
```

Comment:
One of the items on my long dev to-do list is cleaning up some of the inconsistencies in variable names. It seems that to stay consistent with common DAAC/community syntax, dataset should refer to ICESat-2 and product should refer to ATLXX. Does that jibe with your experience? If so, I will work on making this change throughout the icepyx library, allowing us to ultimately get rid of this line of code (clearly a minor point here, but good motivation).

Reply:
Yes - I think it’s better to use ‘product’ to describe different ATLXX rather than ‘datasets’, see OA API also uses [‘product’](https://openaltimetry.org/data/swagger-ui/#/Public/getLevel3Data) to be consistent with the [NSIDC syntax](https://icesat-2.gsfc.nasa.gov/science/data-products).